makespec/BUILDVERSION

```diff
@@ -1 +1,5 @@
-5281
\ No newline at end of file
+<<<<<<< HEAD
+5305
+=======
+5281
+>>>>>>> 91eb8ee1cb0a5d5e1dc9a838cd6f5b90a483e2b5
```

Comment:
What's this?

Reply:
I don't known anything about this. automaticly?